api/src/main/java/org/openmrs/validator/VisitValidator.java

```diff
@@ -107,6 +110,20 @@ public void validate(Object target, Errors errors) {
 				validateStopDatetime(visit, otherVisit, errors);
 			}
 		}
+        	if(visit.getStartDatetime()!=null && visit.getPatient()!=null && visit.getPatient().getBirthdate()!=null)
+        	{
+            		Date birthDateTime = visit.getPatient().getBirthdate();
+            		Date startDateTime = visit.getStartDatetime();
+            		if(startDateTime.compareTo(birthDateTime) < 0) {
```

Comment:
@djazayeri 
I totally agree on that point but i think the birthdate and startdate are always stored in same format
So i think Date.before() or Date.after() will help more maybe

Please correct me if I'm wrong
Thank You :)

My apologies that i didn't notice your comment